test-cases/tests/documentation/i05-managing-users.md

```diff
@@ -0,0 +1,30 @@
+---
```

Comment:
is this a duplicate of `test-cases/tests/documentation/i04-managing-users.md`?

Reply:
nope. It is a different test cast.